README.md

```diff
@@ -62,6 +63,8 @@ Launch in [Pangeo Binder](https://pangeo-binder.readthedocs.io) (Interactive jup
 
 [![Binder](https://binder.pangeo.io/badge_logo.svg)](https://binder.pangeo.io/v2/gh/weiji14/deepbedmap/master)
 
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/gshanbhag525/deepbedmap/blob/master/data_prep.ipynb)
```

Comment:
```suggestion
[![Binder](https://binder.pangeo.io/badge_logo.svg)](https://binder.pangeo.io/v2/gh/weiji14/deepbedmap/master)
[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/weiji14/deepbedmap/blob/master/deepbedmap.ipynb)
```

Let's change it to open `deepbedmap.ipynb` since that's the main Jupyter notebook. Also might be a good idea to point it to the upstream master branch instead.